trinity/nodes/base.py

```diff
@@ -61,15 +96,15 @@ def make_ipc_server(self) -> IPCServer:
         self._ipc_server, self._ipc_loop = ipc_server, ipc_loop
 
         try:
-            asyncio.ensure_future(self._peer_pool.run())
             asyncio.run_coroutine_threadsafe(ipc_server.run(loop=ipc_loop), loop=ipc_loop)
-            await self._peer_chain.run()
+
+            async with RunningServices(self._auxiliary_services):
```

Comment:
It feels like the `self._auxiliary_services` list should get locked down once things begin running.  Otherwise we could end up in a weird situation where a new service is added after things are running.